sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/package-info.java

```diff
@@ -1,16 +1,564 @@
+ * can be used to analyze unstructured text for tasks, such as sentiment analysis, entities recognition(PII, Health,
+ * Linked, Custom), key phrases extraction, language detection, abstractive and extractive summarizations,
+ * single-label and multi-label classifications, and execute multiple actions/operations in a single request.
+ * </p>
+ *
+ * <h2>Getting Started</h2>
+ *
+ * <p>In order to interact with the Text Analytics features in Azure AI Language Service, you'll need to create an
+ * instance of the Text Analytics Client class. To make this possible you'll need the key credential of the service.
+ * Alternatively, you can use AAD authentication via
+ * <a href="https://learn.microsoft.com/java/api/overview/azure/identity-readme?view=azure-java-stable">Azure Identity</a>
+ * to connect to the service.</p>
+ * <ol>
+ *   <li>Azure Key Credential, see {@link com.azure.ai.textanalytics.TextAnalyticsClientBuilder#credential(
+ *   com.azure.core.credential.AzureKeyCredential) AzureKeyCredential}.</li>
+ *   <li>Azure Active Directory, see {@link com.azure.ai.textanalytics.TextAnalyticsClientBuilder#credential(
+ *   com.azure.core.credential.TokenCredential) TokenCredential}.</li>
+ * </ol>
+ *
+ * <p><strong>Sample: Construct Synchronous Text Analytics Client with Azure Key Credential</strong></p>
+ *
+ * <p>The following code sample demonstrates the creation of a {@link com.azure.ai.textanalytics.TextAnalyticsClient},
+ * using the {@link com.azure.ai.textanalytics.TextAnalyticsClientBuilder} to configure it with a key credential.</p>
+ *
+ * <!-- src_embed com.azure.ai.textanalytics.TextAnalyticsClient.instantiation -->
+ * <pre>
+ * TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder&#40;&#41;
+ *     .credential&#40;new AzureKeyCredential&#40;&quot;&#123;key&#125;&quot;&#41;&#41;
+ *     .endpoint&#40;&quot;&#123;endpoint&#125;&quot;&#41;
+ *     .buildClient&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.ai.textanalytics.TextAnalyticsClient.instantiation -->
+ *
+ * <p><strong>Sample: Construct Asynchronous Text Analytics Client with Azure Key Credential</strong></p>
+ *
+ * <p>The following code sample demonstrates the creation of a {@link com.azure.ai.textanalytics.TextAnalyticsAsyncClient},
+ * using the {@link com.azure.ai.textanalytics.TextAnalyticsClientBuilder} to configure it with a key credential.</p>
+ *
+ * <!-- src_embed com.azure.ai.textanalytics.TextAnalyticsAsyncClient.instantiation  -->
+ * <pre>
+ * TextAnalyticsAsyncClient textAnalyticsAsyncClient = new TextAnalyticsClientBuilder&#40;&#41;
+ *     .credential&#40;new AzureKeyCredential&#40;&quot;&#123;key&#125;&quot;&#41;&#41;
+ *     .endpoint&#40;&quot;&#123;endpoint&#125;&quot;&#41;
+ *     .buildAsyncClient&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.ai.textanalytics.TextAnalyticsAsyncClient.instantiation  -->
+ *
+ * <p>See methods in client level class below to explore all features that library provides.</p>
```

Comment:
<p><Strong>Note:</Strong> See methods in client level class below to explore all features that library provides.</p>